resources/js/airline.js

```diff
@@ -0,0 +1,112 @@
+let submitButton = document.getElementById("submitButton");
+const updateButton = document.getElementById("updateButton");
+const hiddenInput = document.getElementById("hidden");
+
+const base_url = import.meta.env.VITE_BASE_URL;
+
+const createAirline = async () => {
+    let name = document.getElementById("name").value;
+    let description = document.getElementById("description").value;
```

Comment:
```suggestion
    const name = document.getElementById("name").value;
    const description = document.getElementById("description").value;
```
When variables don't change, use `const` instead of `let`.